content/en/docs/guides/user/kubernetes-v2/deploy-helm/index.md

```diff
@@ -55,10 +55,12 @@ metadata:
 * __Helm chart file path__ (optional)
 
   Helm chart file path is only relevant (and visible) when the template artifact
-  is a git/repo artifact.  It specifies the path to Chart.yaml in the git repo.
+  is a git/repo artifact.  It specifies the directory path to Chart.yaml in the git repo.
   If absent, spinnaker looks for Chart.yaml in the root directory of the git
   repo.
 
+  Example: If Chart.yaml is a directory named helm `root/helm/` then `chartFilePath` would be `helm/`
```

Comment:
Is this really correct?  For `root/helm/Chart.yaml` woudn't helmChartFilePath be `root/helm` ?  Maybe it's the use of `root` that's throwing me off.  For me, if a path starts with a slash, that slash means root...So maybe this becomes...
```suggestion
  Example: If Chart.yaml is in a directory named helm  (i,e.`/helm/Chart.yaml`) then `helmChartFilePath` would be `/helm`
```
but is the presence (or absence) of the leading slash the thing that was causing you trouble?

Reply:
Hrmm, how about this:
Example
```
Give a git repo where your Chart.yaml is in:
  sub/folder/Chart.yml

```
then you'd want:
```
helmChartFilePath: "sub/folder/"
```

Reply:
Sure.  Does a leading slash break things?

Reply:
@dbyron-sf I agree with on that. I'm afraid someone would actually use the leading slash in the file path, which actualy doesnt work.

Reply:
So perhaps include a note about "no leading slash" ...

Reply:
Tossed my approval in now for what it's worth ;) (I don't have real approval permissions :P )